apps/dot_ext/templates/oauth2_provider/application_detail.html

```diff
@@ -94,7 +107,7 @@ <h2 class="ds-u-margin-bottom--2 ds-u-margin-top--4">App Details - Optional Info
                   <img src="{% static 'img/default-app-avatar.png' %}" alt="{{ application.name }}" />
                   {% endif %}
             
-                  <!-- App TOS/PP-->
+gtgtgt
```

Comment:
Was this intentional?

Reply:
@njdister I noticed that too and forgot to ask @johnfrenchxyz about it today. From https://github.com/CMSgov/bluebutton-web-server/pull/811/files#diff-4b1ece3e47a4e21c9eaabf26ccd56361R106

Reply:
@njdister @dtisza1 - great catch. Not intentional. Looks like i tried to switch tabs in Vim while I was in edit mode and didn't notice the change. We can just remove the `gtgtgt`